external-crates/move/crates/move-core-types/src/resolver.rs

```diff
@@ -55,6 +60,8 @@ pub trait ModuleResolver {
     type Error: Debug;
 
     fn get_module(&self, id: &ModuleId) -> Result<Option<Vec<u8>>, Self::Error>;
+
```

Comment:
The answer is probably `no`, but would it make sense to make a new trait called `PackageResolver` that has `get_package` so that we don't have to add `get_package` to all of the execution versions that already exist? Then the new resolver would be `PackageResolver + ModuleResolver` or something?